src/components/FeaturePanel/Runways/SuccessRunways.tsx

```diff
@@ -0,0 +1,65 @@
+import Link from 'next/link';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@mui/material';
+import React from 'react';
+import { Runway as RunwayType } from './loadRunways';
+import { t } from '../../../services/intl';
+
+const fmtRunwaySize = (runway: RunwayType) =>
+  `${runway.length || '?'} - ${runway.width || '?'}`;
+
+const Runway: React.FC<{ runway: RunwayType; showSize: boolean }> = ({
+  runway,
+  showSize,
+}) => (
+  <TableRow>
+    <TableCell>
+      <Link href={`/${runway.type}/${runway.id}`}>
+        {runway.ref || 'Unknown'}
+      </Link>
+    </TableCell>
+    {showSize && <TableCell>{fmtRunwaySize(runway)}</TableCell>}
+    {/* TODO: Use id tagging presets to show the surface in the users language */}
+    <TableCell style={{ textTransform: 'capitalize' }}>
+      {runway.surface || 'Unknown'}
+    </TableCell>
+  </TableRow>
+);
+
+interface RunwayProps {
+  runways: RunwayType[];
+}
+
+export const SuccessRunways: React.FC<RunwayProps> = ({ runways }) => {
+  const showSize = runways.every(({ length, width }) => length || width);
```

Comment:
Can you please rewrite it without iteration? Why did you use it in the first place?

Reply:
I guess you mean the decleration of `showSize`? This way the size is only shown when at least one runway has a known size. There are many airports where not even a single runway size is mapped.

Reply:
I see, I understood it wrong. Maybe better name could be `showSizeColumn`?

But even now, I don't think it is correct. It would hide the column when there eg. one runway with length and the other without, right? The correct function would be `runways.some()`, right?